integration-cli/docker_cli_build_test.go

```diff
@@ -6632,3 +6632,20 @@ func (s *DockerSuite) TestBuildCacheRootSource(c *check.C) {
 
 	c.Assert(out, checker.Not(checker.Contains), "Using cache")
 }
+
+func (s *DockerSuite) TestBuildFailsGitNotCallable(c *check.C) {
+	path := os.Getenv("PATH")
+	// empty out PATH so git isn't callable
+	c.Assert(os.Setenv("PATH", ""), checker.IsNil)
+
+	out, _, err := dockerCmdWithError("build", "github.com/docker/v1.10-migrator.git")
```

Comment:
why not set `exec.Cmd.Env` ? Not sure what the consequences are of setting PATH via os.Setenv